src/Rules/Install.hs

```diff
@@ -193,6 +193,20 @@ installPackages = do
                 let ghcCabalInplace = inplaceBinPath -/- "ghc-cabal" -- HACK?
                 need [ ghcCabalInplace ]
 
+                let cabalFile = pkgCabalFile pkg
+                -- HsColour sources
+                -- QUESTION: what is the output of GhcCabalHsColour?
+                whenM (isSpecified HsColour) $ do
+                    top <- interpretInContext context getTopDirectory
+                    let installDistDir = top -/- buildPath context
+                    -- HACK: copy stuff back to the place favored by ghc-cabal
+                    quietly $ copyDirectoryContents (Not excluded)
```

Comment:
Oh, right. I didn't realise `GhcCabalHsColour` was broken in the normal build. It may be worth checking what it does and whether we can just run `hscolour` on resulting files ourselves.